### ISSUE TITLE
loss is "nan"

### ISSUE THREAD

**Issue by HaynesLi** [wilile26811249/CMT#6]
Actually, in your code, even the first training loss is nan. How could you "loss.backward()"? Besides, the output of your net is also "nan".

**Comment by Sugarwine99** [comment]
因为他他妈optimizer.zero_grad()这个玩意放错位置了，ctmd

**Comment by wilile26811249** [comment]
Sorry for replying so late and for the error training pipeline.
I have already updated the training pipeline.

**Comment by gongshuai0606** [comment]
Hi, I have the same problem,the first output of the net is 'nan', how can I solve it?
